src/app/app.component.scss

```diff
@@ -54,100 +162,45 @@
                     }
                 }
             }
+            .right-section {
+                width: 100%;
+                display: flex;
+                flex-direction: column;
```

Comment:
move the common styles (you can see an example in the company repo) to the global styles.scss to avoid code duplication